packages/client/src/pages/pinia.vue

```diff
@@ -20,7 +21,12 @@ function getPiniaState(nodeId: string) {
   })
 }
 
+function clearPiniaState() {
+  state.value = {}
+}
+
 watch(selected, () => {
+  clearPiniaState()
```

Comment:
Clear the current state is critical as getPiniaState is an async process.

Prevent using the current select key to render the previous data.